docs/docs/actively-maintained-versions.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: actively-maintained-versions
+sidebar_label: Actively Maintained Versions
+title: Actively Maintained Versions
+abstract: This page gathers information about Rasa Open Source releases that are actively maintained.
+---
+
+We're actively maintaining _any minor on our latest major release_ and _the latest minor of the previous major release_.
+Currently, this means the following minor versions will receive bugfixes updates:
+- 1.10
+- 2.0
+- 2.1
+- 2.2
+- 2.3
+- 2.4
+- 2.5
+- 2.6
+- 2.7
+- 2.8
```

Comment:
I'm worried that we'll forget to update this as we release new minors - can we not just put a 2.x, rather then listing them all explicitly?

Reply:
even if we mention it explicitly in the instructions to prepare a release? I thought that more explicit content would be better for users

Reply:
well I guess the question is whether people actually read those instructions (I don't most of the time because I have done it 1000 times). Is there a way to just automate this?

Reply:
yeah there's a way sure 😅 but I won't have the time to do it this week. Let's pick your solution and see if we want the full list or not later. It's a good first iteration. I'll update the PR